docker/android-system.sh

```diff
@@ -116,49 +559,264 @@ EOF
     rm bionic/tests/headers/Android.bp
     rm bionic/tests/headers/posix/Android.bp
 
-    sed -i -z -e 's/cc_test {.*}//g' bionic/libc/malloc_debug/Android.bp
-    sed -i -z -e 's/cc_test {.*}//g' bionic/libc/malloc_hooks/Android.bp
-    sed -i -z -e 's/cc_test_host {.*}//g' bionic/tools/relocation_packer/Android.bp
```

Comment:
These are all handled by the Python Soong parser below, and have the added benefit of actually handling matching braces. Before, the sed script would either need to be greedy (so these would remove contents until the end of the file), or non-greedy, in which case no inner objects could exist.